db/db_test_util.cc

```diff
@@ -72,7 +72,13 @@ DBTestBase::~DBTestBase() {
   options.db_paths.emplace_back(dbname_ + "_2", 0);
   options.db_paths.emplace_back(dbname_ + "_3", 0);
   options.db_paths.emplace_back(dbname_ + "_4", 0);
-  EXPECT_OK(DestroyDB(dbname_, options));
+
+  const char* keep_db = getenv("KEEP_DB");
+  if (keep_db && keep_db[0] == '1') {
```

Comment:
if future using only the first char of an env might cause confusions. how about using it as a bool the same way we already interpreting USE_MEM:
```
(!getenv("MEM_ENV") 
```